packages/devtools-reps/src/object-inspector/index.js

```diff
@@ -74,28 +77,6 @@ type State = {
   focusedItem: ?ObjectInspectorItem
 };
 
-type ObjectInspectorItemContentsValue = {
-  actor: string,
-  class: string,
-  displayClass: string,
-  extensible: boolean,
-  frozen: boolean,
-  ownPropertyLength: number,
-  preview: Object,
-  sealed: boolean,
-  type: string,
-};
-
-type ObjectInspectorItemContents = {
-  value: ObjectInspectorItemContentsValue,
-};
-
-type ObjectInspectorItem = {
```

Comment:
haha - didn't realize they were here already. Still though, would be nice to shorten